app/code/Magento/Analytics/Model/Config/Backend/Enabled.php

```diff
@@ -67,12 +67,7 @@ public function afterSave()
         try {
             if ($this->isValueChanged()) {
                 $enabled = $this->getData('value');
-
-                if ($enabled) {
-                    $this->subscriptionHandler->processEnabled();
-                } else {
-                    $this->subscriptionHandler->processDisabled();
-                }
+                $enabled ? $this->subscriptionHandler->processEnabled() : $this->subscriptionHandler->processDisabled();
```

Comment:
Ternary operator is NOT really suitable here as it decreases code readability. it is better to be used in assignments only.